tools/devctr/Dockerfile.x86_64

```diff
@@ -4,8 +4,7 @@ FROM ubuntu:18.04
 # The Rust toolchain layer will get updated most frequently, but we could keep the system
 # dependencies layer intact for much longer.
 
-ARG RUST_TOOLCHAIN="1.43.1"
-ARG RUST_TOOLCHAIN_AUDIT="1.47.0"
```

Comment:
Does this change mean auditing will always be done with the latest stable version (instead of the previously hard-coded value)?

Reply:
yass